ceph_iscsi_config/lun.py

```diff
@@ -574,6 +577,44 @@ def activate(self):
             if client_err:
                 raise CephiSCSIError(client_err)
 
+    def _erasure_pool_check(self):
+        # skip it and if no ecpool specified
+        if not self.datapool:
+            return True
```

Comment:
A data pool doesn't have to be an EC pool.  A user may want to create an image with a separate replicated pool.  So two things:
- `_erasure_pool_check()` should check the pool type and, only if the pool is EC, check whether `allow_ec_overwrites` is enabled
- The objectstore check should be removed.  As I already noted in https://github.com/ceph/ceph-iscsi/pull/237#discussion_r669708186, enabling `allow_ec_overwrites` on any object store other than `bluestore` isn't possible in normal configurations so `allow_ec_overwrites` check would be sufficient.